client/src/components/fedbackPopUp/controller.js

```diff
@@ -0,0 +1,33 @@
+function fedbackPopUpController($http, $scope, $element) {
+  let self = this;
+  self.$element = $element;
+
+  $scope.$parent.openPopUpClick = function(id){
+    let idLecture = '#fb'+id;
```

Comment:
Not good idea to set property to the parent from one of its children. Imaging what will be if you will have two popups declared in parent template. The second one will overwrite $parent.openPopupClick method.